lib/screens/account.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+import 'package:scoped_model/scoped_model.dart';
+import 'package:ofypets_mobile_app/scoped-models/main.dart';
+import 'package:ofypets_mobile_app/screens/auth.dart';
+import 'package:ofypets_mobile_app/screens/order_history.dart';
+import 'package:ofypets_mobile_app/screens/change_email.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+import 'package:ofypets_mobile_app/utils/constants.dart';
+import 'package:http/http.dart' as http;
+
+class Account extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text("Account"),
+      ),
+      body: Container(
+        child: account_options(),
+      ),
+    );
+  }
+
+  Widget account_options() {
```

Comment:
Naming convention for  non-constant identifiers is lowerCamelCase. So please try to follow this convention and avoid underscores.